pkg/sources/git/git.go

```diff
@@ -319,6 +321,43 @@ type cloneParams struct {
 	clonePath string
 }
 
+//Defines the interface for returning a single PID value given an executable name
+type PID interface {
+	getPIDs(ctx context.Context, executable string) (int, error)
+}
+
+var ExecutableName = "trufflehog"
+
+func GetScannerPIDs(ctx context.Context, executable string) (int, error)  {
+	var pids []int
+
+	procs, err := ps.Processes()
+	if err != nil {
+		return 0, fmt.Errorf("error getting jobs PIDs: %w", err)
+	}
+
+	for _, proc := range procs {
+		if strings.Contains(proc.Executable(), executable) {
+			pids = append(pids, proc.Pid())
+		}
+	}
+
+	if len(pids) < 2 {
+		return pids[0], nil
+	} else {
+
+	        // Sort the PIDs in ascending order
+	        sort.Ints(pids)
+
+	        // Return the lowest value of the first two PIDs
+	        if pids[0] < pids[1] {
+	        	return pids[0], nil
+	        }
+	        return pids[1], nil
```

Comment:
I don't understand why we're explicitly comparing the first two in the list if it's already sorted? Wouldn't `pids[0]` always be the lowest value?